dask_kubernetes/core.py

```diff
@@ -203,7 +203,7 @@ def __init__(self, **kwargs):
         # TODO Resolve service race condidion
         # There is some race condition happening here where the service isn't actually ready by the time
         # this function exits.
-        await asyncio.sleep(2)
+        await asyncio.sleep(5)
```

Comment:
Oof, that's a long sleep

Reply:
It sure is! I've been cranking it up from `0.1` and I've only now managed to get the tests to pass. 

I've been digging into the k8s API to try and find something that I can watch for a state change to avoid this sleep, but so far no luck.

Reply:
Is this the problem that we were running into over screenshare?  

if it's a major pain we can always try https://github.com/dask/distributed/issues/3084

Reply:
Yeah. I'm trying to solve it cleanly, but 3084 may be a good workaround.